TESTING.md

```diff
@@ -98,7 +98,7 @@ the tests.
 These tests can be run by providing a system property.  Specific ITs can be run using "-Dit.test" or run all tests using:
 
 ```bash
-mvn clean verify -Dtest=foo -Daccumulo.it.properties=/home/user/my_cluster.properties -Dfailsafe.groups=org.apache.accumulo.test.categories.StandaloneCapableClusterTests -Dspotbugs.skip
+mvn clean verify -Dtest=foo -Daccumulo.it.properties=/home/user/my_cluster.properties -Dfailsafe.groups=StandaloneCapableClusterTests -Dspotbugs.skip
```

Comment:
These tag names can be simplified further. The original names like `SunnyDayTests` was chosen because we had to create a class name that made sense on its own. Now that's not necessary, since the class is deleted, and the fact that we're in a "Test" context is already obvious when using the Tag annotation. So, something like `@Tag("SunnyDay")` should suffice instead of `@Tag("SunnyDayTests")`.

Reply:
Good idea. I'm wondering if in that case we could remove the [`-Psunny` profile](https://github.com/apache/accumulo/blob/97af14509d2755de29e2c0196201018f576cab68/pom.xml#L1548-L1555) since to run the same tests now, it would just be `-Dfailsafe.groups=SunnyDay`. The prior is still shorter but just thought I would point it out and see what you/others thought. I assume the profile was created since this group of tests is run often, and before, one would have to type out the full class name (org.apache.accumulo.test.categories.SunnyDayTests).

Reply:
I agree it is superfluous, but it's historical, so it's probably worth keeping. When we initially added these category annotations, I resisted creating a bunch of new profiles with different combinations of activating them, because you can just use the `-Dfailsafe.groups=` property. I still think it's unnecessary to create new profiles, but keeping this one around for historical reasons is probably desirable. If you wanted to simplify it further, you could call it `@Tag("Sunny")` in order to get even closer to the profile name.

Reply:
Addressed in [d1fa1f3](https://github.com/apache/accumulo/pull/2545/commits/d1fa1f31dbbf5e8728ceaa9359f2e92b67cd4d75)